modules/nw-external-dns-operator.adoc

```diff
@@ -0,0 +1,104 @@
+----
+$ oc get -n external-dns-operator deployment/external-dns-operator
+NAME                    READY     UP-TO-DATE   AVAILABLE   AGE
+external-dns-operator   1/1       1            1           23h
+----
+
+. Use the `oc get` command to view the state of the External DNS Operator:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get subscription external-dns-operator -o yaml | yq e '.status'
+----
++
+.Example output
+[source,yaml]
+----
+catalogHealth:
+  - catalogSourceRef:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      name: certified-operators
+      namespace: openshift-marketplace
+      resourceVersion: "76597"
+      uid: dbee97d8-6f98-4d80-a644-bbeb29311c05
+    healthy: true
+    lastUpdated: "2022-05-09T11:50:15Z"
+  - catalogSourceRef:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      name: community-operators
+      namespace: openshift-marketplace
+      resourceVersion: "79665"
+      uid: eeb546ff-7e0a-4aa0-a1d5-d1a477a9b738
+    healthy: true
+    lastUpdated: "2022-05-09T11:50:15Z"
+  - catalogSourceRef:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      name: external-dns-operator
+      namespace: openshift-marketplace
+      resourceVersion: "76595"
+      uid: d374f646-3191-47c1-9b5b-7aa8e94f2405
+    healthy: true
+    lastUpdated: "2022-05-09T11:50:15Z"
+  - catalogSourceRef:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      name: redhat-marketplace
+      namespace: openshift-marketplace
+      resourceVersion: "76539"
+      uid: 98f8c536-adb4-4c4a-8852-d2ba889197f0
+    healthy: true
+    lastUpdated: "2022-05-09T11:50:15Z"
+  - catalogSourceRef:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      name: redhat-operators
+      namespace: openshift-marketplace
+      resourceVersion: "77843"
+      uid: 2781ad03-f16d-4df4-83af-66144e9e8b87
+    healthy: true
+    lastUpdated: "2022-05-09T11:50:15Z"
+conditions:
+  - lastTransitionTime: "2022-05-09T11:50:15Z"
+    message: all available catalogsources are healthy
+    reason: AllCatalogSourcesHealthy
+    status: "False"
+    type: CatalogSourcesUnhealthy
+currentCSV: external-dns-operator.v1.0.0
+installPlanGeneration: 1
+installPlanRef:
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: InstallPlan
+  name: install-nrv7w
+  namespace: external-dns-operator
+  resourceVersion: "80282"
+  uid: 399ef3c3-d242-4abc-b203-165728ac664f
+installedCSV: external-dns-operator.v1.0.0
+installplan:
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: InstallPlan
+  name: install-nrv7w
+  uuid: 399ef3c3-d242-4abc-b203-165728ac664f
+lastUpdated: "2022-05-11T09:50:08Z"
+state: AtLatestKnown
+----
++
+`AVAILABLE`, `PROGRESSING` and `DEGRADED` provide information about the status of the Operator. `AVAILABLE` is `True` when at least 1 pod from the `ExternalDNS` daemon set reports an Available status condition.
```

Comment:
This isn't true with ExternalDNS Operator as it's installed via OLM, the check of `installplan` before is sufficient.

Reply:
Ack'ed